ChemistryLib/CreateChemicalSolverInterface.cpp

```diff
@@ -186,9 +187,15 @@ createChemicalSolverInterface<ChemicalSolver::PhreeqcKernel>(
         //! \ogs_file_param{prj__chemical_system__rates}
         config.getConfigSubtreeOptional("rates"));
 
+    // pure phase assemblage
```

Comment:
Easier name would be collection. (If you do mean it and not "assembly".)

Reply:
The name comes from phreeqc, but we should make the nomenclature compatible with OGS.
What about "phase components"

Reply:
This object consists of not only `pure_phases` (`phase_components`) but also other members. That's why I feel object naming is too difficult...

Reply:
If you specify me which other members you are making reference, may I can help, total mass?

Reply:
What we call `pure_phase_assemblage` at this moment consists of the members listed below:
`new_def` -> a kind of flag;
`pp_assemblage_comps` -> `pure_phases` (`phase_components`);
`eltList`, -> list of elements in `pure_phases`;
`assemblage_totals` -> total moles of elements;
`vopts` -> ...

`pp_assemblage_comps` is a set of `pure_phase` which holds phase attributes such as:
`name` -> phase name;
`si` -> saturation index
`moles` and so on

Reply:
How about naming this assemblage as `EquilibirumPhases`. We call the member `pp_assemblage_comps` `phase_components`. @montoyav @endJunction

Reply:
OK, I understand, all the functionalities that the keyword EQUILIBRIUM_PHASES has in PHREEQC... let me think....

Reply:
I was having a look to all the lines, to make it consistent:
1) EquilibriumPhases as @renchao-lu suggested or
2) EquilibriumReactants

Reply:
So `EquilibriumReactants` for the entire object and `PhaseComponents` for the member `pp_assemblage_comps`.